src/modules/services/submodule/mainnetSwap/components/SelectInputAccountOutgoingSwap.vue

```diff
@@ -163,7 +164,7 @@ export default defineComponent({
       }
       selectedAccountBalance.value = otherToken.value!='prx.xpx'? 
       accounts.find(account => account.address == accountAddress).assets.find(asset=>asset.namespaceNames==otherToken.value)? 
-      accounts.find(account => account.address == accountAddress).assets.find(asset=>asset.namespaceNames==otherToken.value).amount/Math.pow(10,6) : 0 
+      accounts.find(account => account.address == accountAddress).assets.find(asset=>asset.namespaceNames==otherToken.value).amount/Math.pow(10,divisibility.value) : 0 
       :accounts.find(account => account.address == accountAddress).balance
       selectedAccountBalance2.value = otherToken.value!='prx.xpx'? accounts.find(account => account.address == accountAddress).balance :0
```

Comment:
selectedAccountBalance and selectedAccountBalance2, maybe change to proper naming ? `selectedAccountTokenBalance`, `selectedAccountNativeBalance`